runtime/etc/Makefile.mli-static

```diff
@@ -23,15 +23,31 @@ include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.include
 
 CHPL_MLI_INCS = -I$(CHPL_ZMQ_LOC)/include
 CHPL_LN_LIB_DIR = $(CHPL_MAKE_RUNTIME_LIB)/$(CHPL_MAKE_LAUNCHER_SUBDIR)
+UNAME = $(shell uname)
+CHPL_LIBTOOL = libtool
+CHPL_LIBTOOL_CMD =
 
-.PHONY: buildUserObj buildServer buildClient runTagsCommand
+.PHONY: setLibtoolCmd buildUserObj buildServer buildClient runTagsCommand
 
-all: buildUserObj buildServer buildClient runTagsCommand FORCE
+all: setLibtoolCmd buildUserObj buildServer buildClient runTagsCommand FORCE
+
+#
+# On some platforms (Darwin in particular), libtool cannot be assumed to mean
+# GNU libtool and is actually a different executable that goes by the same
+# name. Darwin's libtool has similar semantics to GNU libtool, but has to be
+# invoked differently.
+#
+setLibtool:
+ifeq ($(UNAME),Darwin)
+	CHPL_LIBTOOL_CMD += $(CHPL_LIBTOOL) -static -no_warning_for_no_symbols
+else
+	CHPL_LIBTOOL_CMD += $(CHPL_LIBTOOL) --quiet --mode=link $(CC) -static
```

Comment:
What output gets generated when it is not `--quiet`?  Are we potentially going to miss something important?

Reply:
I was trying to find this out myself, GNU libtool also has a `--no-warn` flag, so it makes me think that it distinguishes between normal output and warnings.

If I could figure out a testcase, though...

Reply:
Eh, if it isn't to silence a particular set of error messages, I would just leave it off and let libtool tell us when we're doing something warning/error-worthy